newIDE/app/src/SceneEditor/ScenePropertiesDialog.js

```diff
@@ -201,7 +201,7 @@ export default class ScenePropertiesDialog extends Component<Props, State> {
             label={<Trans>Open advanced settings</Trans>}
             fullWidth
             onClick={() => {
-              this.props.onOpenMoreSettings();
+              this.props.onOpenMoreSettings && this.props.onOpenMoreSettings();
```

Comment:
Prefer a `if` than a && for readability.

Reply:
Done.